apps/src/applab/themeColor.js

```diff
@@ -77,7 +77,7 @@ module.exports = {
     polar: '#ACCFDE',
     glowInTheDark: '#4D575F',
     bubblegum: '#FFD4ED',
-    millennial: 'rgba(0, 0, 0, 0)',
+    millennial: '#FEDBD0',
```

Comment:
All of the hard-coded strings in this file should be constants with reader-friendly names

Reply:
If they're only used here I'm not sure I'd expect them to be pulled out as standalone constants. Where would you expect these to be defined?

Reply:
yes, this is essentially the constants file

Reply:
Since so many of these are repeated between themes (especially colors like white), a personal preference of mine would be to move them to reader-friendly names. Not needed though.